pipeline/console/container/__init__.py

```diff
@@ -509,22 +514,22 @@ def _push_container(namespace: Namespace):
             sys.stdout.write(print_string)
             sys.stdout.flush()
 
+    _print("Successfully pushed image to registry")
+
     new_deployment_request = http.post(
         endpoint="/v4/pipelines",
-        json_data=json.loads(
-            pipelines_schemas.PipelineCreate(
-                name=pipeline_name,
-                image=image_to_push_reg,
-                input_variables=[],
-                output_variables=[],
-                accelerators=pipeline_config.accelerators,
-                description=pipeline_config.description,
-                readme=pipeline_config.readme,
-                extras=pipeline_config.extras,
-                cluster=pipeline_config.cluster,
-                scaling_config=pipeline_config.scaling_config_name,
-            ).json()
-        ),
+        json_data=pipelines_schemas.PipelineCreate(
+            name=pipeline_name,
+            image=image_to_push_reg,
+            input_variables=[],
+            output_variables=[],
+            accelerators=pipeline_config.accelerators,
+            description=pipeline_config.description,
+            readme=pipeline_config.readme,
+            extras=pipeline_config.extras,
+            cluster=pipeline_config.cluster,
+            scaling_config=pipeline_config.scaling_config_name,
+        ).dict(),
```

Comment:
lol, that's nicer

Reply:
yeah not sure why it was like it was before!